R/trim_ps_table.R

```diff
@@ -30,6 +30,9 @@
 #' }
 #'
 #' @importFrom dplyr select all_of
+#' 
+#' @examples trim_ps_table(ps_data) # Trim data from planetary systems table 
```

Comment:
issue, style: No need for one-lining this, also the comment is redundant.
```suggestion
#' @examples
#' trim_ps_table(ps_data)
```